code/edgemedia/src/phone/java/com/adobe/marketing/mobile/edge/media/Media.java

```diff
@@ -122,13 +122,13 @@ private Media() {}
     }
 
     /**
-     * Creates an instance of the QoS info object
+     * Creates an instance of the QoE info object
      *
      * @param bitrate The bitrate of media in bits per second
      * @param startupTime The start up time of media in seconds
      * @param fps The current frames per second information
      * @param droppedFrames The number of dropped frames so far
-     * @return A MediaObject instance representing the QoSObject.
+     * @return A MediaObject instance representing the QoEObject.
```

Comment:
Let's update the all the API's return statements with something like:
`@return A map representing Quality of experience information` which we have in our [swift documentation](https://github.com/adobe/aepsdk-edgemedia-ios/blob/dev/Documentation/api-reference.md#createqoeobjectwith) 

If you agree with this change we should update our Android docs create* API docs as well.